README.md

```diff
@@ -27,9 +30,17 @@ API Definition
 --------------
 
 See the human-readable [Reference Documentation](https://ga4gh.github.io/task-execution-schemas/docs/)
-and the [OpenAPI YAML description](openapi/task_execution_service.openapi.yaml). You can also explore the specification in the [Swagger Editor](https://editor.swagger.io/?url=https://ga4gh.github.io/task-execution-schemas/openapi.yaml).
+You can also explore the specification in
+the [Swagger Editor](https://editor.swagger.io/?url=https://ga4gh.github.io/task-execution-schemas/openapi.yaml)**
+*Manually load the JSON if working from a non-develop branch version.* Preview documentation from
```

Comment:
I think there is some markdown mixup here. See: https://github.com/ga4gh/task-execution-schemas/blob/develop/README.md